common/test/keyboards/full_caps_3620_3621/source/full_caps_3620_3621.kmn

```diff
@@ -1,6 +1,6 @@
 ﻿c full_caps_3620_3621 generated from template at 2021-12-01 08:45:15
 c with name "full_caps_3620_3621"
-store(&VERSION) '10.0'
+store(&VERSION) '15.0'
```

Comment:
Why this change? Is this another test case for this PR, or does this keyboard use features introduced in 15.0?

We'll have to adjust this for stable-14.

Reply:
The U_xxxx_yyyy is a 15.0 feature which will cause the 14.0 Keyman app to crash trying to render the popup keys.
I presume the Windows build is now failing due to a test. will chase that down.

Reply:
The CI fail is cause all the common/test/keyboards are expected to build so it's wanting the keyboard project...

I didn't check it in cause I had to manually edit the built kmp.json file...

Reply:
I am about to add a child PR to this branch that fixes the build issue for common/test/keyboards

Reply:
See #6358